content/docs/next-release/operator.md

```diff
@@ -0,0 +1,645 @@
+---
+title: Operators
+hasparent: true
+weight:
+---
+
+# Understanding Operators
+
+The Jaeger Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/). It provides a method of packaging, deploying, and managing Jaeger.
+
+Operators are pieces of software that ease the operational complexity of running another piece of software. More technically, _Operators_ are a method of packaging, deploying, and managing a Kubernetes application.
+
+A Kubernetes application is an app that is both deployed on Kubernetes and managed using the Kubernetes APIs and `kubectl` (kubernetes) or `oc` (OKD) tooling. To be able to make the most of Kubernetes, you need a set of cohesive APIs to extend in order to service and manage your apps that run on Kubernetes. Think of Operators as the runtime that manages this type of app on Kubernetes.
+
+
+# Installing the Operator
+
+IMPORTANT: The Jaeger Operator version is related to the version of the Jaeger components (Query, Collector, Agent) up to the minor portion. The patch version portion does *not* follow the ones from the Jaeger components. For instance, the Operator version 1.8.1 uses the Jaeger Docker images tagged with version 1.8 by default.
```

Comment:
Markdown has no support for admonitions. We might need some other formatting here.

Reply:
I went with bold.

Reply:
I'll add some admonition capabilities. PR coming later today 👍

Reply:
See #261

Reply:
@lucperkins   I saw this right after I pushed an update.  I'll take a look at #261 tomorrow.  Thanks!